src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -64,6 +59,7 @@ public record VectorSearchConfig(
                 implementation = Integer.class)
             @JsonProperty("dimension")
             @JsonAlias("size") // old name
+            @NotNull
```

Comment:
Based on new spec this can be null, if supported vectorize algorithm is provided. We need to populate it with dimension from the properties.

Reply:
captured in https://datastax.jira.com/jira/software/c/projects/C2/boards/525?assignee=712020%3Ac496b02d-1ca9-4999-845b-3b7b5cf1bc17&selectedIssue=C2-3314